cmd/create/oidcconfig/cmd.go

```diff
@@ -207,7 +207,8 @@ func run(cmd *cobra.Command, argv []string) {
 					"to be compliant with OIDC protocol. It will also create a Secret in Secrets Manager containing the private key")
 			}
 			if mode == aws.ModeAuto && (interactive.Enabled() || (confirm.Yes() && args.installerRoleArn == "")) {
-				args.installerRoleArn = interactive.GetInstallerRoleArn(r, cmd, args.installerRoleArn, minorVersionForGetSecret)
+				args.installerRoleArn = interactive.GetInstallerRoleArn(r, cmd, args.installerRoleArn,
+					minorVersionForGetSecret, false)
```

Comment:
@gdbranco do you know what should be the value here? For `false` it will filter out non-HCP managed policies

Reply:
hmm, we should be able to show either here, it only cares that the installer role has GetSecretValue permission

Reply:
It was changed to ignore the policy type for creating the OIDC config.